cli/run.sh

```diff
@@ -0,0 +1 @@
+python -m cli.text2embeddings --s3 $parser_output_s3 $indexer_output_s3
```

Comment:
if you merge this branch from main you can add the `--device=cuda` flag in here but also happy to approve that in another pr :)

Reply:
Integrated in this pr now :)